dash/backend/src/modules/falco/controllers/falco.controller.ts

```diff
@@ -88,27 +93,51 @@ export class FalcoController {
 
 
     @Post(':clusterid/create')
-    // @TODO: Update to use same authorization method as kube-bench and kube-hunter
     async createFalcoLog(
         @Param('clusterid') clusterId: number,
         @Body() falcoLog: FalcoWebhookInputDto,
         @Query('key') key: string
-    ): Promise<number> {
-
-        //find user authority by user's apikey
+    ): Promise<FalcoDto> {
+        // look up user by api key
         const currentUserAuthObj = await this.userDao.loadUserByApiKey(key);
-        // if the current user's api is valid
-        if (currentUserAuthObj !== null && currentUserAuthObj != undefined ) {
-            // get all authorities from the current user
-            const currentUserAuth = currentUserAuthObj[0].authorities;
-            // get Falco authority from Authorityid
-            let authorityArr: AuthorityId [] = [AuthorityId.FALCO];
-            // is the user a Falco user
-            let isFalcoUser = this.authService.checkAuthority(currentUserAuth, authorityArr);
-            if (isFalcoUser) {
-                return this.falcoService.createFalcoLog(clusterId, falcoLog);
-            }
+        if (!currentUserAuthObj) {
+            return; // @TODO: Should throw access denied exception
```

Comment:
Should throw UnauthorizedException if access credentials are invalid

See https://github.com/m9sweeper/m9sweeper/blob/main/dash/backend/src/modules/auth/services/ldap/generic-ldap.service.ts#L92 as an example.